pulp_container/tests/functional/api/test_pull_content.py

```diff
@@ -135,6 +138,32 @@ def test_api_returns_same_checksum(self):
             "Cannot find a matching layer on remote registry.",
         )
 
+    def test_api_performes_schema_conversion(self):
+        """Verify pull via token with accepted content type.
+        """
+        image_path = "/v2/{}/manifests/{}".format(self.distribution_with_repo.base_path, "latest")
+        latest_image_url = urljoin(self.cfg.get_base_url(), image_path)
+
+        with self.assertRaises(requests.HTTPError) as cm:
+            self.client.get(latest_image_url, headers={"Accept": MEDIA_TYPE.MANIFEST_V1})
+
+        content_response = cm.exception.response
+        self.assertEqual(content_response.status_code, 401)
+
+        authenticate_header = content_response.headers["Www-Authenticate"]
+        queries = AuthenticationHeaderQueries(authenticate_header)
+        content_response = self.client.get(
+            queries.realm, params={"service": queries.service, "scope": queries.scope}
+        )
+        token = content_response.json()["token"]
+        content_response = self.client.get(
+            latest_image_url,
+            auth=BearerTokenAuth(token),
+            headers={"Accept": MEDIA_TYPE.MANIFEST_V1},
+        )
+        base_content_type = content_response.headers["Content-Type"].split(";")[0]
+        self.assertIn(base_content_type, {MEDIA_TYPE.MANIFEST_V1, MEDIA_TYPE.MANIFEST_V1_SIGNED})
```

Comment:
Seems like we are returning `'application/vnd.docker.distribution.manifest.v1+prettyjws; charset=utf-8'` when asking for `'application/vnd.docker.distribution.manifest.v1+json'`. Is this intended?

Reply:
We probably shouldn't be doing that unless we are actually signing the manifest. Let's file a bug about it.

Reply:
https://pulp.plan.io/issues/7113